### ISSUE TITLE
fm_manifold ignores all type options but the first given (0.1.6)

### ISSUE THREAD

**Issue by finnlindgren** [finnlindgren/fmesher#16]
Test:
```r
fm_manifold(fm_rcdt_2d_inla(globe = 2), type = c("R", "S"))
# FALSE
```
Expected result:
```r
# TRUE
```

This is due to a regression where it only considers a single type option.

Reported in https://github.com/eliaskrainski/INLAspacetime/issues/3

**Comment by finnlindgren** [comment]
Fixed in 0.1.6.9001

**Comment by finnlindgren** [comment]
Version 0.1.7 that fixes this is now on its way onto CRAN
